frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -206,13 +206,74 @@ common_settings.document = {
             },
         },
         {
-            text = _("Show book status at end of document "),
-            checked_func = function()
-                return G_reader_settings:nilOrTrue("auto_book_status")
-            end,
-            callback = function()
-                G_reader_settings:flipNilOrTrue("auto_book_status")
-            end,
+            text = _("End of document action"),
+            sub_item_table = {
+                {
+                    text = _("Ask with pop-up dialog"),
+                    checked_func = function()
+                        local setting = G_reader_settings:readSetting("end_document_action")
+                        return setting == "pop-up" or setting == nil
+                    end,
+                    callback = function()
+                        G_reader_settings:saveSetting("end_document_action", "pop-up")
+                    end,
+                },
+                {
+                    text = _("Do nothing"),
+                    checked_func = function()
+                        return G_reader_settings:readSetting("end_document_action")
+                            == "nothing"
```

Comment:
I'm not sure what I think of the legibility of this so also pingin @poire-z @NiLuJe ;-)

Reply:
Could use `false`, or `"disable"`, as in:
https://github.com/koreader/koreader/blob/e3b7524d9c85ac810f8084d538df2ef3bbf8e2c7/frontend/ui/elements/screensaver_menu.lua#L86-L96

Reply:
It would be ever so slightly faster I suppose but I was talking about the line split. :-P

Reply:
Oh, ok :) yes, that line starting with `==` is strange; could be made on one line (we have longer lines than that elsewhere).

Reply:
FWIW, I tend to go with 120 characters as a max width, since that matches GitHub's own code viewer ;). (And, also, the tiny screen on my laptop :D).

So, yeah, we could defiinitively avoid the split.

Reply:
Alright, please avoid the line splits. :-)